### ISSUE TITLE
Reported Result for Vit-large

### ISSUE THREAD

**Issue by jytime** [pengzhiliang/MAE-pytorch#27]
Hi, 

I notice the reported perfromance of vit-large is 84.5%. I was wondering, does this correspond to the baseline of the paper?

In other words, can we say the reproduced v.s. official is 84.5% : 84.9%?

Thanks

**Comment by pengzhiliang** [comment]
Hello,

The guy pre-trains vit-large for 400e, and fine-tune it for 50e, with target normalization,  and reach 84.5%.
The baseline pre-trains vit-large for 1600e, and fine-tune it for 50e, without target normalization,  and reach 84.9%.

All in all, the setting is different. I think the gap may not be too big even if it can't be fully reproduced.
